dist/js/foundation.js

```diff
@@ -1323,7 +1323,7 @@
 	});
 })(jQuery);
 /****************************************************
- * Method for adding psuedo drag events to elements *
+ * Method for adding pseudo drag events to elements *
```

Comment:
Dist files are automatically generated 
You should be editing source files instead 

Here are the files/lines where you should edit,
=> https://github.com/zurb/foundation-sites/blob/develop/js/foundation.util.touch.js#L79
=> https://github.com/zurb/foundation-sites/blob/develop/js/foundation.sticky.js#L108
=> https://github.com/zurb/foundation-sites/blob/develop/js/foundation.sticky.js#L149